libs/kotaemon/kotaemon/indices/vectorindex.py

```diff
@@ -22,7 +23,7 @@ class VectorIndexing(BaseIndexing):
         - List of documents
         - List of texts
     """
-
+    cache_dir: Optional[str] = flowsettings.KH_CHUNKS_OUTPUT_DIR
```

Comment:
If you access `KH_CHUNKS_OUTPUT_DIR` directly from `flowsettings`, it will raise error if the user doesn't have this KH_CHUNKS_OUTPUT_DIR in their flowsettings. So instead, you should change it to:

```
getattr(flowsettings, "KH_CHUNKS_OUTPUT_DIR", None)
```